rbac/enforcer.go

```diff
@@ -110,6 +110,7 @@ func Init(kapi etcd.KeysAPI) error {
 		return err
 	}
 	loadConfigPolicy(model)
+	casbinEnforcer.BuildRoleLinks()
```

Comment:
Is it required ?

Reply:
yes otherwise panic do to not having the RoleManager set